### DISCUSSION TITLE
debug status=500

### DISCUSSION THREAD

**Discussion by conloos** [smallstep/certificates#1197] [General]
Hi @all,

I have built a Smallstep CA with v0.23.0 (Jumping from 0.18.1) and now I get an error 500 with the ACME client "dehydrated".
In the log I find the error:
`Dec 03 12:52:32 ca.example.com step-ca[1259]: time="2022-12-03T12:52:32+01:00" level=error duration=41.409926ms duration-ns=41409926 error="not found" fields. time="2022-12-03T12:52:32+01:00" method=POST name=ca nonce=bjc2RVB3bm44aUVvcmZtQXFTUFV1U0ljWEZmb3N2Zks path=/acme/acme/new-order protocol=HTTP/1. 1 referer= remote-address=192.168.178.1 request-id=ce5jh00voc6tauif17kg size=3 status=500 user-agent="dehydrated/0.7.0 curl/7.84.0" user-id=`

I tryed to activate debug in step-ca.service (https://smallstep.com/docs/step-cli/the-step-command):
`Environment=STEPDEBUG=1`

But I still do not get more information.

Is there an article on how to proceed here?

Thanks Frank

**Comment by hslatman** [comment]
Is it possible the CA was started with a fresh database? Or did you upgrade and kept using the old database? Can you try running dehydrated from a fresh state, e.g. not having an account registered with the CA yet?
